api/src/wfl/util.clj

```diff
@@ -377,3 +379,34 @@
   [parts]
   (letfn [(make-part [[k v]] {:name (name k) :content v})]
     (map make-part parts)))
+
+(defn str->date
+  "Parse date string `s` to date object, with optional `format`."
+  ([s format]
+   (.parse (SimpleDateFormat. format) s))
+  ([s]
+   (str->date s "yyyy-MM-dd")))
+
+(defn datetime->str
+  "Parse `datetime` object into string, with optional `format`.
+
+   Note this function uses UTC as the time zone offset by default."
+  ([^Instant datetime format]
+   (let [formatter (-> format
+                       DateTimeFormatter/ofPattern)]
+     (.format formatter datetime)))
+  ([^Instant datetime]
+   (datetime->str datetime "yyyy-MM-dd")))
```

Comment:
You don't need this as we discussed? Just use `toString()` of `LocalDate`?

Reply:
```clojure
(format "%s" (java.time.LocalDate/now))
=> "2021-03-17"
```

Reply:
But you're not using it for that - you're using it to get the date, not an instant?

Reply:
AH, I think I over-complicated it. Lemme switch to LocalDate with UTC…

Reply:
Updated per offline discussion, use `LocalDate` as we don't need DateTime yet.